fit/README.md

```diff
@@ -0,0 +1,61 @@
+# Fit file as available from Strava
+
+Pull your own data from your Strava account. Visualize your data, manipulate your data, combine your data with other sources,  all while maintaining full ownership and control of the data in your own environment.
+
+## Table of contents
+
+ * `ThursMorn.fit` - the data used to run this example
+ * `accessFit.py` - python script to demonstrate fit data inside deephaven 
+ * `*.png` - images used in this help
```

Comment:
```suggestion
 * `*.png` - Images used in this help.
```

Reply:
In this help? or in these examples?

Reply:
I defer to whatever you / your team decides as the best final home for this chunk of work.